### ISSUE TITLE
Repl.it won't auto include it

### ISSUE THREAD

**Issue by PotentialStyx** [Codemonkey51/replit-db-client#3]
On repl the package manager won't auto detect it as a package that needs to be imported

**Comment by frissyn** [comment]
This is an issue with repl.it's universal package manager. It probably won't be able to auto-detect until replitdb has been on PyPi for an extended period of time.

**Comment by PotentialStyx** [comment]
Okay I've labeled it so it says that I can't fix already

**Comment by PotentialStyx** [comment]
> This is an issue with repl.it's universal package manager. It probably won't be able to auto-detect until replitdb has been on PyPi for an extended period of time.

i talked to turbio and he said only certian popular packages are added and replitdb fits that, but they dont update the index often, but they may soon

TLDR: turbio says it will be added soon
